UMLEditor/UMLEditor/Views/MainWindow.axaml.cs

```diff
@@ -834,6 +878,16 @@ private void ViewEditToggle_OnClick(object sender, RoutedEventArgs e)
             #pragma warning restore CS8629
         }
 
+        private void MagicLinesToggle_OnClick(object sender, RoutedEventArgs e)
+        {
+            ToggleSwitch viewSwitch = (ToggleSwitch) sender;
+
+            RelationshipLine.ToggleMagicLines(((bool)viewSwitch.IsChecked));
+            
```

Comment:
We should be looking at cleaning up the warnings. More warnings have been introduced than were before

Reply:
Warnings have been taken care of.